src/components/chip-set/chip.types.ts

```diff
@@ -89,3 +89,5 @@ export interface Chip<T = any> {
      */
     href?: string;
 }
+
+export type ChipType = 'default' | 'filter';
```

Comment:
This needs a docblock.

```suggestion
/**
 * This type is used to determine the visual style and behavior of a Chip component.
 *
 * @beta
 */
export type ChipType = 'default' | 'filter';
```

Reply:
do we really need a `@beta` tag here? I know it's new and shiny, but I don't think we need it for this private component ;) :P

Reply:
We do and we don't. We don't technically _need_ it, as the PR-checks will be green anyway. And we get this little warning in the api-extractor output:

<img width="1079" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/154725/1e5fa71b-2f52-42bf-a536-e3618894081f">

That is as it should be, because it should warn us about that. But, the problem is that when you later make the component and type public, and you perhaps forget to add a docblock to the type, there is no update in the api-extractor output, which means the maintainer reviewing it will not be notified that there should be a docblock added.

And the type is visible in the docs, even if you consider it internal:

<img width="695" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/154725/969365cf-c48c-4ee7-8bcc-f284623b905a">

If it's meant to be internal permanently, not just while the `limel-chip` component is private, it should be marked with `@internal`.

Reply:
I can't really answer the question. These things are really confusing for me. I trust you do what is right ;)